.github/workflows/test.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           load: true
       - name: Run Perl tests
         run: >
-          docker run -d -i -p 8000:80
+          docker run -d -i -p 8000:8000
```

Comment:
I think these may need to be changed to `5001`? I just merged that change to the port number. 😢